### PR TITLE
Simplify the example on the landing page

### PR DESCRIPTION
**Pull request by mshubham00**
calling COF algorithm as a function

### CODE REVIEW

File: easy-example-experiment.py [hunk 1]
```diff
@@ -1,46 +1,37 @@
 #!/usr/bin/env python3
 from pathlib import Path
 from typing import Any, Dict
-
 import numpy as np
 
 from timeeval import TimeEval, DatasetManager, DefaultMetrics, Algorithm, TrainingType, InputDimensionality
-from timeeval.adapters import DockerAdapter, FunctionAdapter
-from timeeval.algorithms import cof
+from timeeval.adapters import DockerAdapter
```
Comment (CodeLionX) [inline easy-example-experiment.py:7]: Not needed anymore, remove.

File: easy-example-experiment.py [hunk 2]
```diff
@@ -1,46 +1,37 @@
...
+from timeeval.adapters import FunctionAdapter # for defining customized algorithm
+from timeeval.algorithms import cof 
 from timeeval.data_types import AlgorithmParameter
 from timeeval.params import FixedParameters
 
-
 def your_algorithm_function(data: AlgorithmParameter, args: Dict[str, Any]) -> np.ndarray:
     if isinstance(data, np.ndarray):
         return np.zeros_like(data)
-    else:  # isinstance(data, pathlib.Path)
```
Comment (CodeLionX) [inline easy-example-experiment.py:17]: why did you remove this?

File: easy-example-experiment.py [hunk 3]
```diff
@@ -1,46 +1,37 @@
...
+    else:
         return np.genfromtxt(data, delimiter=",", skip_header=1)[:, 1]
 
-
 def main():
     dm = DatasetManager(Path("tests/example_data"), create_if_missing=False)
-    datasets = dm.select()
-
+    datasets = dm.select()    
+
```
Comment (CodeLionX) [inline easy-example-experiment.py:22]: Please avoid whitespace changes and remove this change.

File: easy-example-experiment.py [hunk 4]
```diff
@@ -1,46 +1,37 @@
...
-            name="COF",
-            main=DockerAdapter(image_name="ghcr.io/timeeval/cof", skip_pull=True),
-            param_config=FixedParameters({
-                "n_neighbors": 20,
-                "random_state": 42
-            }),
-            data_as_file=True,
-            training_type=TrainingType.UNSUPERVISED,
-            input_dimensionality=InputDimensionality.MULTIVARIATE
+    # list of algorithms which will be implimented on the selected dataset(s)
```
Comment (CodeLionX) [inline easy-example-experiment.py:24]:
```suggestion
        # list of algorithms, which will be executed on the selected dataset(s)
```

**Review by CodeLionX — CHANGES_REQUESTED** [review]

File: easy-example-experiment.py (hunk 2 above)
Comment (mshubham00) [reply easy-example-experiment.py:17]: it was commented code line, not in use. 

I thought it was unnecessary and not useful for first time user

Comment (CodeLionX) [reply easy-example-experiment.py:17]: I documents the type of the `else`-branch. Please add it back in.

**Review by CodeLionX — APPROVED** [review]
